Lobby/EOSLobby.cs

```diff
@@ -0,0 +1,197 @@
+﻿using EpicTransport;
+using Epic.OnlineServices.Lobby;
+using UnityEngine;
+using Epic.OnlineServices;
+using System.Collections.Generic;
+
+public class EOSLobby : MonoBehaviour
+{
+    [HideInInspector] public bool ConnectedToLobby = false;
+    public LobbyDetails ConnectedLobbyDetails = new LobbyDetails();
```

Comment:
This should be private

Reply:
I disagree. Having access to this allows for users' UI implementation to have access to data attached to that lobby (ex. member count)

Reply:
Then make it a property with public get private set to prevent outside changes